.github/workflows/submitBeta.yml

```diff
@@ -0,0 +1,122 @@
+name: Beta Deployment
+on:
+  workflow_dispatch:
+    inputs:
+      release:
+        description: Release Number
+        default: "2.0.0"
+        required: true
+      version:
+        description: Beta version
+        default: "0"
+        required: true
+jobs:
+  bump-version:
+    name: Bump Package Version and Submit Extension
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+        with:
+          ref: release/${{ github.event.inputs.release }}
+      - name: Update manifest.json name
+        uses: jossef/action-set-json-field@6e6d7e639f24b3955ef682815317b5613ac6ca12 #v1
+        with:
+          file: ./extension/public/static/manifest.json
+          field: name
+          value: Freighter BETA
+      - name: Update manifest.json version_name
+        uses: jossef/action-set-json-field@6e6d7e639f24b3955ef682815317b5613ac6ca12 #v1
+        with:
+          file: ./extension/public/static/manifest.json
+          field: version_name
+          value: ${{ github.event.inputs.release }}-beta.${{ github.event.inputs.version }}
+      - name: Update manifest.json version for Chrome
+        uses: jossef/action-set-json-field@6e6d7e639f24b3955ef682815317b5613ac6ca12 #v1
+        with:
+          file: ./extension/public/static/manifest.json
+          field: version
+          value: ${{ github.event.inputs.release }}.${{ github.event.inputs.version }}
+      - name: Build extension
+        uses: actions/setup-node@v1
+        with:
+          node-version: "12"
+      - run: yarn install && yarn build:freighter-api && yarn build:extension:production
+      - name: Install zip
+        uses: montudor/action-zip@c25e01d7489d0274569440a2f0281b4569df16bc #v0.1.1
+      - name: Zip extension build
+        run: zip -qq -r ./build.zip *
+        working-directory: ./extension/build
+      # - name: Create GitHub Release
```

Comment:
is this commented out code still needed?

Reply:
Yeah, I'm keeping this disabled for now because I need to confirm that just the "Submit extension to Chrome" bit works before I enable the whole pipeline.

I'm testing some new creds and didn't want to deal with creating new Firefox builds and releases if the whole thing was going to ultimately fail at "Submit extension to Chrome"